subgraphs/synapse/README.md

```diff
@@ -0,0 +1,105 @@
+# Synapse 
+
+Synapse is a cross-chain liquidity network secured by validators utilizing multi-party computation (MPC) and threshold signature schemes (TSS). It combines AMMs from multiple chains with its own liquidity pools to allow efficient and low slippage transfer of assets between chains. All in all, Synapse allows users to effortlessly and seamlessly bridge or swap assets between chains, with over $10.5 billion USD in bridged volume. 
+
+## Usage Metrics
+
+`Active Users`, `Total Unique Users` & `Daily Transaction Count`
+
+Users of Synapse are those taking the following actions:
+
+Bridge Assets
+Swap Assets
+Liquidity Pools
+Deposit Assets
+Withdraw Assets
+Stake LP position
+
+
+## Total Value Locked USD
+
+Synapse operates by utilizing multiple liquidity pools across many different AMMs. Liquidity providers of these AMMs can stake their LP position with Synapse, to help facilitate cross chain swaps and bridging in return for SYN emissions. 
```

Comment:
Hmm, I think they have their own AMM. They don’t use third parties’ as far as I know?

Reply:
updated to indicate correction: Synapse operates as an AMM utilizing multiple liquidity pools across many different chains. Liquidity providers of the AMM can stake their LP position with Synapse, to help facilitate cross chain swaps and bridging in return for SYN emissions.